modules/local/generax.nf

```diff
@@ -40,5 +41,12 @@ process GENERAX {
     --per-family-rates \
     --strategy SPR \
     --prefix GeneRax
+    
+    # Rename the inferred reconciled gene trees to be named after their corresponding orthogroup
```

Comment:
I bet there's a non-loop version of this but I am not sure, so I'll let this slide!

Reply:
I suspect so too, just gets tricky since they're all in their own directories, each of which is created from the single call of the generax program.